python_training/sim_city.py

```diff
@@ -0,0 +1,207 @@
+BASE_MUNICIPALITY_TAX_RATE = 1000
+
+
+class Ackland:
+    """
+    The state Ackland itself.
+    """
+
+    def __init__(self, cities):
```

Comment:
Add type hints to c'tors, since you're supporting multiple input options.
Here, for example:
`def __init__(self, cities: Union[City, List[City]]):`